src/platform-includes/enriching-events/attach-screenshots/android.mdx

```diff
@@ -3,3 +3,43 @@
   <meta-data android:name="io.sentry.attach-screenshot" android:value="true" />
 </application>
 ```
+
+### Customize Screenshot Capturing
+
+<Note>
+
+Requires SDK version `6.24.0` or higher.
+
+</Note>
+
+On Android capturing screenshots can be an resource-intense operation, thus screenshot capturing is limited to one screenshot every 2 seconds using a debouncing mechanism. This behavior can be overruled by supplying a `BeforeCaptureCallback` for screenshots in the `SentryAndroidOptions`.
```

Comment:
```suggestion
Because capturing screenshots can be a resource-intensive operation on Android, it's limited to one screenshot every 2 seconds using a debouncing mechanism. This behavior can be overruled if you supply a `BeforeCaptureCallback` for screenshots in the `SentryAndroidOptions`.
```